### ISSUE TITLE
Default config startup failed in case RDT is enabled only with MBM

### ISSUE THREAD

**Issue by kad** [intel/cri-resource-manager#41]
Supported schemata:
```
$ cat /sys/fs/resctrl/schemata 
MB:0=100;1=100;2=100;3=100
$
```

```
I: [resource-manager] using auto-discovered resctrl-path "/sys/fs/resctrl"
D: [       rdt      ] writing schemata "L3:0=1;1=1;2=1;3=1\nMB:0=100;1=100;2=100;3=100\n"
E: [   cri-resmgr   ] failed to create resource manager instance: resource-manager: failed to create resource manager: rdt: configuration failed: write /sys/fs/resctrl/cri-resmgr.BestEffort/schemata: invalid argument
```

**Comment by marquiz** [comment]
Would/should be fixed by #37

**Comment by marquiz** [comment]
Fixed in #37
